src/client/components/forms/relationship.js

```diff
@@ -188,7 +243,12 @@ class RelationshipEditor extends React.Component {
 			});
 		}
 	}
-
+	/**
+	* This takes in a row index, finds out the updated relationships,
```

Comment:
Gets the most current information about relationships in the form, and saves it in the form state, to update the state following a change and propagate the change to the displayed UI.